projects/view/client/singleProject/singleProjectDetailed.js

```diff
@@ -0,0 +1,156 @@
+  		if(this.type=='facebook') {
+  			return 'fa fa-facebook';
+  		}
+  		if(this.type=='twitter') {
+  			return 'fa fa-twitter';
+  		}
+  		if(this.type=='instagram') {
+  			return 'fa fa-instagram';
+  		}
+  		//need to implement font awesome icon for blogger and other link types used
+  		if(this.type=='blogger') {
+  			return 'fa fa-blogger';
+  		}
+  		if(this.type=='linkedin') {
+  			return 'fa fa-linkedin';
+  		}
+  	},
+    startupDateFormatted: function () {
+      return moment(this.startupDate).format('YYYY');
+    },
+
+    websites: function () {
+
+      var websites = jQuery.grep(this.links, function( n, i ) {
+        return ( n.type === 'web' );
+      });
+      return websites;
+    },
+
+    socialLinks: function () {
+
+      var websites = jQuery.grep(this.links, function( n, i ) {
+        return ( n.type === 'web' ||
+          n.type === 'facebook' ||
+          n.type ===  'twitter' ||
+          n.type === 'instagram' ||
+          n.type === 'blogger' ||
+          n.type === 'linkedin'
+        );
+      });
+      return websites;
+    },
+
+    hasMultipleWebsites: function () {
+
+      //TODO: Duplicated code, can we reuse instead?
+      var websites = jQuery.grep(this.links, function( n, i ) {
+        return ( n.type === 'web' );
+      });
+
+      return websites.length > 1;
+    },
+    isWebsite: function () {
+      return this.type=='web';
+    },
+    calculateProgress: function(){
+
+      switch (this.currentStage) {
+        case "initiation":
+          return 10;
+          break;
+        case "planning":
+          return 20;
+          break;
+        case "implementationExecution":
+          return 50;
+          break;
+        case "operationMonitoring":
+          return 70;
+          break;
+        case "closing":
+          return 90;
+          break;
+        default:
+          return 0;
+      }
+    },
+    projectMedia: function () {
+      return ProjectMedia.find({_id: this.mediaId});
+    }
+});
+
+Template.singleProjectDetailed.events({
+    'click#show-more ': function(event,template) {
+    	$('.content-text').addClass('visable');
+		$(event).target.css({
+			display:'none',
+		});
+    },
+    "click#view-images": function(event,template) {
+		if($(event).target.hasClass('active')){
+			$('.image_slide_wrapper').removeClass('image_slide_wrapper_active');
+			$('#view-images').removeClass('active');
+			$('.projectDetailList').removeClass('inactive');
+		}
+		else{
+			$('.image_slide_wrapper').addClass('image_slide_wrapper_active');
+			$('#view-images').addClass('active');
+			$('.projectDetailList').addClass('inactive');	
+		}
+    },
+    "click.projectDetailList":function(event,template) {
+		$('.image_slide_wrapper').removeClass('image_slide_wrapper_active');
+		$('#view-images').removeClass('active');
+		$('.projectDetailList').removeClass('inactive');
+    },
+    
+    
+})
+
+Template.singleProjectDetailed.onRendered(function () {
```

Comment:
Move these jQuery events into the `Template.singleProjectDetailed.events({})` map.